README.md

```diff
@@ -100,13 +108,55 @@ adding an amplifier. *Only go in this direction after familiarizing yourself
 with allowances of radio transmissions in your area on your frequency of
 interest.*
 
-### Transmit!
+#### Nvidia Jetson Series (experimental)
+(*Please read the external hardware for Raspberry Pi above first.)   
+On Jetson, the external hardware setup is slightly different from Raspberry Pi. 
+
+We use hardware PWM pin as a frequency output pin. Unlike the Raspberry Pi, we need only one output pin 
+because we modulate the signal by switching (on-off keying) instead of attenuating.
```

Comment:
Note, some of the signals require attenuation to be in spec. So if you want to skip implement attenuation in the first step, maybe point out that it might not work in all circumstances.